oslabs-beta/GraphPulse
# Bm/resolvers

## Overview

**Issue Type**


**Description**

updated index.js file and GraphQL server setup
created mock data, resolvers and typeDefs files. 
 
**Ticket/Card Name**

Scrum 9 - Set upGraphQl API   
	
**Steps to Reproduce/Validate**

1. Checkout BM/resolvers branch
2. Navigate to `index.js`, `mockData.js`, `resolvers.js`, and `typeDefs.js`
3. Run the GraphQL server using Apollo Server 
4. Access the GraphQL sandbox
5. Execute queries -> expected data retrieval

**Expected behavior**

Queries are executed in the sandbox and return expected data based on the mock data provided.

**Screenshots & Videos**
N/A

**Additional context**

NA
